src/app/record/components/work-stack/work-stack.component.ts

```diff
@@ -134,5 +137,12 @@ export class WorkStackComponent implements OnInit {
     return item.putCode.value
   }
 
+  userIsSource(work: Work): boolean {
+    if (this.userInfo) {
+      return work.source === this.userInfo.REAL_USER_ORCID;
```

Comment:
I believe `REAL_USER_ORCID` represents the logged-in user, so this will be a problem with delegate users.  I think you need to use `EFFECTIVE_USER_ORCID`, please test this first.

Reply:
Check this on affiliations and fundings as well

Reply:
You are right!. I will fix it